backend/app/db/database_service.py

```diff
@@ -68,36 +69,52 @@ def index_media():
     client.index("media").add_documents(medias)
 
 
-async def extract_unique_providers_to_cache():
-    db = database.SessionLocal()
-    db_media = get_all_media(db)
-
-    supported_countries = {x: set() for x in get_settings().supported_country_codes}
-
-    for media in db_media:
-        if media.providers:
-            filtered_countries = [
-                country
-                for country in list(media.providers.keys())
-                if country in supported_countries
-            ]
-            for country_code in filtered_countries:
-                for provider_type in ["flatrate", "free"]:
-                    if providers := media.providers.get(country_code).get(
-                        provider_type
-                    ):
-                        for provider in providers:
-                            supported_countries[country_code].add(
-                                provider["provider_name"]
-                            )
-
-    for country_code in get_settings().supported_country_codes:
-        providers: set = supported_countries[country_code]
-        ordered_providers: list = sorted(providers)
-
+async def providers_to_redis():
+    providers = {}
+    countries_url = (
+        f"{get_settings().tmdb_url}"
+        f"configuration/countries?api_key={get_settings().tmdb_key}"
+    )
+    providers_movie_url = (
+        f"{get_settings().tmdb_url}"
+        f"watch/providers/movie?api_key={get_settings().tmdb_key}"
+    )
+    providers_tv_url = (
+        f"{get_settings().tmdb_url}"
+        f"watch/providers/movie?api_key={get_settings().tmdb_key}"
```

Comment:
You've included movie in both this url and the one above

Reply:
Fixed